django/db/models/sql/compiler.py

```diff
@@ -676,8 +693,14 @@ def find_ordering_name(self, name, opts, alias=None, default_order='ASC',
 
             results = []
             for item in opts.ordering:
-                results.extend(self.find_ordering_name(item, opts, alias,
-                                                       order, already_seen))
+
+                def _join_strings(*strings):
+                    return LOOKUP_SEP.join(filter(None, strings))
```

Comment:
What is this for? Is the `filter()` necessary? Couldn't this just be inlined below?

Reply:
I think this is the alternative
```python
if prefix is None:
    new_prefix = name
else:
    new_prefix = prefix + LOOKUP_SEP + name
```
prefix is `None` at the top of the recursive call.

Which is better?